.github/workflows/pydax.yml

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright 2020 IBM Corp. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: PyDAX
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: 'master'
+  pull_request:
+    branches: '*'
```

Comment:
Is there any benefit to testing PRs? Won't `pydax` be looking at the master branch?

Reply:
The PRs update the data files and this would test whether the changes can pass minimum quality as input to PyDAX

Reply:
I guess what I'm saying is the PR would update this file (until it's merged):
https://raw.githubusercontent.com/CODAIT/dax-schemata/PR-BRANCH-NAME/datasets.yaml

And pydax would be downloading this:
https://raw.githubusercontent.com/CODAIT/dax-schemata/master/datasets.yaml

Reply:
Oh I got what you mean. The script need to be updated.